<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>notepad</class>
 <widget class="QDialog" name="notepad">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>873</width>
    <height>565</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NOTEPAD</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(15, 22, 40);
</string>
  </property>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>350</y>
     <width>581</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:#E5CB94;
</string>
   </property>
   <property name="placeholderText">
    <string>YOUR TEXT HERE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_BACK">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>500</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#pushButton_BACK
{
	border:none;
	background:#0F1628;
	color:#E5CB94;
	border-radius:16px;
	border: 1px solid yellow ;
border-color: #E5CB94;
  border-radius: 12px;
  padding: 3px;
}

#pushButton_BACK:hover
{
	border:none;
	background:floral white;
	color:#0F1628;
	border-radius:16px;
	border: 2px solid blue ;
border-color: #0F1628;
  border-radius: 12px;
  padding: 1px;
}
</string>
   </property>
   <property name="text">
    <string>BACK</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>320</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(229, 203, 148);
</string>
   </property>
   <property name="text">
    <string>Last Updated </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_TIME">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>320</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(229, 203, 148);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget_list">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>60</y>
     <width>731</width>
     <height>201</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background:#E5CB94;
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAsNeeded</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::SelectedClicked</set>
   </property>
   <property name="dragEnabled">
    <bool>false</bool>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::ContiguousSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <property name="verticalScrollMode">
    <enum>QAbstractItemView::ScrollPerItem</enum>
   </property>
   <property name="gridStyle">
    <enum>Qt::SolidLine</enum>
   </property>
   <property name="sortingEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_CLEAR">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>490</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#pushButton_CLEAR
{
	border:none;
	background:#0F1628;
	color:#E5CB94;
	border-radius:16px;
	border: 1px solid yellow ;
border-color: #E5CB94;
  border-radius: 12px;
  padding: 3px;
}

#pushButton_CLEAR:hover
{
	border:none;
	background:floral white;
	color:#0F1628;
	border-radius:16px;
	border: 2px solid blue ;
border-color: #0F1628;
  border-radius: 12px;
  padding: 1px;
}
</string>
   </property>
   <property name="text">
    <string>CLEAR</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonDELETE_ALL">
   <property name="geometry">
    <rect>
     <x>689</x>
     <y>280</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#pushButtonDELETE_ALL
{
	border:none;
	background:#0F1628;
	color:#E5CB94;
	border-radius:16px;
	border: 1px solid yellow ;
border-color: #E5CB94;
  border-radius: 12px;
  padding: 3px;
}

#pushButtonDELETE_ALL:hover
{
	border:none;
	background:floral white;
	color:#0F1628;
	border-radius:16px;
	border: 2px solid blue ;
border-color: #0F1628;
  border-radius: 12px;
  padding: 1px;
}
</string>
   </property>
   <property name="text">
    <string>DELETE ALL</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>20</y>
     <width>761</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(229, 203, 148);
</string>
   </property>
   <property name="text">
    <string>NOTEPAD</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>360</y>
     <width>82</width>
     <height>103</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_ADD">
      <property name="styleSheet">
       <string notr="true">#pushButton_ADD
{
	border:none;
	background:#0F1628;
	color:#E5CB94;
	border-radius:16px;
	border: 1px solid yellow ;
border-color: #E5CB94;
  border-radius: 12px;
  padding: 3px;
}

#pushButton_ADD:hover
{
	border:none;
	background:floral white;
	color:#0F1628;
	border-radius:16px;
	border: 2px solid blue ;
border-color: #0F1628;
  border-radius: 12px;
  padding: 1px;
}
</string>
      </property>
      <property name="text">
       <string>ADD</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_UPDATE">
      <property name="styleSheet">
       <string notr="true">#pushButton_UPDATE
{
	border:none;
	background:#0F1628;
	color:#E5CB94;
	border-radius:16px;
	border: 1px solid yellow ;
border-color: #E5CB94;
  border-radius: 12px;
  padding: 3px;
}

#pushButton_UPDATE:hover
{
	border:none;
	background:floral white;
	color:#0F1628;
	border-radius:16px;
	border: 2px solid blue ;
border-color: #0F1628;
  border-radius: 12px;
  padding: 1px;
}
</string>
      </property>
      <property name="text">
       <string>UPDATE</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_DELETE">
      <property name="styleSheet">
       <string notr="true">#pushButton_DELETE
{
	border:none;
	background:#0F1628;
	color:#E5CB94;
	border-radius:16px;
	border: 1px solid yellow ;
border-color: #E5CB94;
  border-radius: 12px;
  padding: 3px;
}

#pushButton_DELETE:hover
{
	border:none;
	background:floral white;
	color:#0F1628;
	border-radius:16px;
	border: 2px solid blue ;
border-color: #0F1628;
  border-radius: 12px;
  padding: 1px;
}
</string>
      </property>
      <property name="text">
       <string>DELETE</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
